src/AutoRest.CSharp/LowLevel/Output/LowLevelClientMethod.cs

```diff
@@ -13,6 +14,10 @@ internal class LowLevelClientMethod : RestClientMethod
     {
         internal class SchemaDocumentation
         {
+            public const string RequestBody = "Request Body";
```

Comment:
Could we replace the use of `IReadOnlyDictionary<string, SchemaDocumentation[]>` and this fixed set of keys with a record that just has three nullable properties, each of type `SchemaDocumentation[]` and then in the writer set these headings?  I think it will be better long term vs using a dictionary where the key is a fixed string that also happens to be used in how we render the documentation.

Reply:
Updated.

Reply:
With your refactoring, are these constants still needed, or can we remove them?

Reply:
They're needed as the name of the schema which is a bit different from the property name.